geeksurvey/templates/home.html

```diff
@@ -13,138 +13,139 @@
 
   <!-- Logic for when a user visits the home page and is authenticated. -->
   <!-- TODO: Update this once a template has been applied to all other pages. -->
-  <body class="d-flex flex-column min-vh-100">
-    {% include 'navbar.html' %}
+  <body>
```

Comment:
Should probably leave in the body class to avoid footer formatting issues:
```body class="d-flex flex-column min-vh-100"```

Also looks like some tab spacing may have been changed in this file.